src/Container/AddWmsPanel/AddWmsLayerEntry/AddWmsLayerEntry.tsx

```diff
@@ -49,6 +52,7 @@ export class AddWmsLayerEntry extends React.Component<AddWmsLayerEntryProps, Add
    */
   constructor(props: AddWmsLayerEntryProps) {
     super(props);
+    // TODO: getAttributions is not @api and returns a function in v6.5
```

Comment:
This needs to be adpated/fixed before we can merge this.

Reply:
this actually has nothing todo with this commit, but was a problem that existed before. Maybe it is only documented is badly. I opened this issue for it: https://github.com/terrestris/react-geo/issues/1962